cypress/support/pages/home.pageObject.js

```diff
@@ -3,9 +3,20 @@ import PageObject from '../PageObject';
 class homePageObject extends PageObject {
   url = '/#/';
 
-  get usernameLink() {
-    return cy.getByDataCy('/profile/');
+  usernameLink(username) {
+    cy.getByDataCy(`/profile/${username}`)
+      .should('contain', username);
   }
-}
+
+  newUsernameLink(username) {
+    cy.getByDataCy(`/profile/${username}`)
```

Comment:
The `newUsernameLink` method seems to duplicate the functionality of the `usernameLink` method. Consider removing it if it's not used, or rename it and adjust its functionality if it is intended to serve a different purpose.